main.py

```diff
@@ -2,6 +2,54 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    # Create
+    genres = ["Western", "Action", "Dramma"]
+    for genre_name in genres:
+        Genre.objects.create(name=genre_name)
+
+    names = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
```

Comment:
I think that "genres" and "names" it is a constant. Use it in global namespace. I think it will be better)))

Reply:
It's unnecessary to use it in the global namespace, 
because these lists affect only the main function, so it's more logical to keep them here